ios/RNZeroconf/RNZeroconf.m

```diff
@@ -207,11 +207,7 @@ - (instancetype) init
 
 - (void) reportError:(NSDictionary *)errorDict
 {
-    for (int a = 0; a < errorDict.count; ++a) {
-        NSString *key = [[errorDict allKeys] objectAtIndex:a];
-        NSString *val = [errorDict objectForKey:key];
-        [self.bridge.eventDispatcher sendDeviceEventWithName:@"RNZeroconfError" body:val];
-    }
+    [self.bridge.eventDispatcher sendDeviceEventWithName:@"RNZeroconfError" body:[NSString stringWithFormat:@"%@",errorDict]];
```

Comment:
Will this correctly throw an object?

Reply:
Yes, it will throw an event body is a string. in the listener of the javascript side, it expects a string. so this should be good